packages/frontend/src/pages/instructlab/StartInstructLabContainer.svelte

```diff
@@ -0,0 +1,141 @@
+    containerProviderConnection = startedContainerProviderConnectionInfo[0];
+  }
+});
+
+function processTasks(trackedTasks: Task[]): void {
+  // Check for errors
+  // hint: we do not need to display them as the TasksProgress component will
+  errorMsg = trackedTasks.find(task => task.error)?.error;
+
+  const task: Task | undefined = trackedTasks.find(task => 'containerId' in (task.labels ?? {}));
+  if (task === undefined) return;
+
+  containerId = task.labels?.['containerId'];
+}
+
+// Submit method when the form is valid
+async function submit(): Promise<void> {
+  errorMsg = undefined;
+  try {
+    await instructlabClient.requestCreateInstructlabContainer({
+      connection: $state.snapshot(containerProviderConnection),
+    });
+  } catch (err: unknown) {
+    console.error('Something wrong while trying to create the InstructLab container.', err);
+    errorMsg = String(err);
+  }
+}
+
+// Navigate to the new created service
+function openInstructLabContainer(): void {
+  instructlabClient.routeToInstructLabContainerTerminal(containerId!).catch(console.error);
+}
+
+function openDocumentation(): void {
+  studioClient.openURL('https://docs.instructlab.ai').catch(console.error);
+}
+</script>
+
+<FormPage title="Run InstructLab as a container">
+  <svelte:fragment slot="content">
+    <div class="flex flex-col w-full">
+      <header class="mx-5 mt-5">
+        <div class="w-full flex flex-row">
+          {#if available}
+            <Button inProgress={!available} title="Open InstructLab container" on:click={openInstructLabContainer}>
+              Open InstructLab container
+            </Button>
+          {:else}
+            <Button title="Start InstructLab container" inProgress={loading} on:click={submit}>
+              Start InstructLab container
+            </Button>
+          {/if}
+          <Button title="Read documentation" type="link" on:click={openDocumentation}>Read documentation</Button>
+        </div>
+      </header>
+      <!-- tasks tracked -->
+      <TrackedTasks
+        class="mx-5 mt-5"
+        onChange={processTasks}
+        trackingId={INSTRUCTLAB_CONTAINER_TRACKINGID}
+        tasks={$tasks} />
+
+      <!-- form -->
+      <div class="bg-[var(--pd-content-card-bg)] m-5 space-y-6 px-8 sm:pb-6 xl:pb-8 rounded-lg h-fit">
+        <div class="w-full text-[var(--pd-details-body-text)]">
+          <!-- container provider connection input -->
+          {#if startedContainerProviderConnectionInfo.length > 1}
+            <label for="" class="pt-4 block mb-2 font-bold text-[var(--pd-content-card-header-text)]"
+              >Container engine</label>
+            <ContainerProviderConnectionSelect
+              bind:value={containerProviderConnection}
+              containerProviderConnections={startedContainerProviderConnectionInfo} />
+          {/if}
+
+          <h1 class="pt-4 mb-2 text-lg first-letter:uppercase">Instructions</h1>
+
```

Comment:
Is there a way to add more details or links to redirect the user to the container's detail page?

Alternatively, I'd propose a bit more guidance:

> Once InstructLab container is started from Podman Desktop, you can start using and experimenting with it by directly getting a terminal into the container. For that, go to the list of containers and search for InstructLab container, click to get into the details of that container. Find the "terminal" tab, where you can input the following commands.
> 
> Please check the documentation to learn more about these commands and how InstructLab works.

Reply:
If the container is started then the button is Open InstructLab container instead of Start